src/components/DropZone/components/FileUpload/FileUpload.scss

```diff
@@ -9,6 +9,17 @@ $slim-vertical-padding: ($slim-min-height - line-height(body) - rem(2px)) / 2;
   text-align: center;
 }
 
+.FileUploadSmallView {
+  padding: 1.25rem;
```

Comment:
It looks like this might not be vertically aligned?

![image](https://screenshot.click/Build_14546_-_Shopify_-_Percy__Visual_testing_as_a_service_2020-08-27_16-44-35.png)

Hard to from the Percy screenshot